scripts/lib/cluster.sh

```diff
@@ -14,23 +14,21 @@ function down-test-cluster() {
 function up-test-cluster() {
     echo -n "Configuring cluster $CLUSTER_NAME"
     ssh-keygen -q -P cni-test -f $SSH_KEY_PATH
-    AWS_K8S_TESTER_EKS_CLUSTER_NAME=$CLUSTER_NAME \
+
+    AWS_K8S_TESTER_EKS_NAME=$CLUSTER_NAME \
         AWS_K8S_TESTER_EKS_KUBECONFIG_PATH=$KUBECONFIG_PATH \
-        AWS_K8S_TESTER_EKS_KUBERNETES_VERSION=${K8S_VERSION%.*} \
-        AWS_K8S_TESTER_EKS_ENABLE_MANAGED_NODE_GROUP_PRIVILEGED_PORT_ACCESS=true \
-        AWS_K8S_TESTER_EKS_MANAGED_NODE_GROUP_ASG_MIN=3 \
-        AWS_K8S_TESTER_EKS_MANAGED_NODE_GROUP_ASG_MAX=3 \
-        AWS_K8S_TESTER_EKS_MANAGED_NODE_GROUP_ASG_DESIRED_CAPACITY=3 \
-        AWS_K8S_TESTER_EKS_MANAGED_NODE_GROUP_REMOTE_ACCESS_PRIVATE_KEY_PATH=$SSH_KEY_PATH \
-        AWS_K8S_TESTER_EKS_MANAGED_NODE_GROUP_INSTANCE_TYPES=m3.xlarge \
-        AWS_K8S_TESTER_EKS_AWS_K8S_TESTER_PATH=$TESTER_PATH \
+        AWS_K8S_TESTER_EKS_PARAMETERS_VERSION=${K8S_VERSION%.*} \
+        AWS_K8S_TESTER_EKS_ADD_ON_MANAGED_NODE_GROUPS_ENABLE=true \
+        AWS_K8S_TESTER_EKS_ADD_ON_MANAGED_NODE_GROUPS_MNGS={\"${CLUSTER_NAME}-mng-for-cni\":{\"name\":\"${CLUSTER_NAME}-mng-for-cni\",\"tags\":{\"group\":\"amazon-vpc-cni-k8s\"},\"ami-type\":\"AL2_x86_64\",\"asg-min-size\":3,\"asg-max-size\":3,\"asg-desired-capacity\":3,\"instance-types\":[\"c5.xlarge\"]}} \
+        AWS_K8S_TESTER_EKS_ADD_ON_MANAGED_NODE_GROUPS_REMOTE_ACCESS_PRIVATE_KEY_PATH=$SSH_KEY_PATH \
         AWS_K8S_TESTER_EKS_AWS_IAM_AUTHENTICATOR_PATH=$AUTHENTICATOR_PATH \
         AWS_K8S_TESTER_EKS_ADD_ON_JOB_ECHO_ENABLE=true \
         AWS_K8S_TESTER_EKS_ADD_ON_JOB_ECHO_PARALLELS=3 \
         AWS_K8S_TESTER_EKS_ADD_ON_JOB_ECHO_COMPLETES=30 \
         AWS_K8S_TESTER_EKS_ADD_ON_JOB_PERL_ENABLE=true \
         AWS_K8S_TESTER_EKS_ADD_ON_NLB_HELLO_WORLD_ENABLE=true \
         AWS_K8S_TESTER_EKS_ADD_ON_ALB_2048_ENABLE=true \
+        AWS_K8S_TESTER_EKS_ADD_ON_SECRETS_ENABLE=false \
```

Comment:
try to keep a PR to a single change of the test config. Otherwise, it's impossible to know which change in isolation is fixing the buggy behaviour.